src/control/server/mgmt_system_test.go

```diff
@@ -2971,6 +3034,520 @@ func TestServer_MgmtSvc_SystemRebuildManage(t *testing.T) {
 
 			test.AssertEqual(t, tc.expCtlApiCount, mi.GetInvokeCount(),
 				"rpc client invoke count")
+
+			if tc.expCtlApiCount > 0 {
+				for _, sr := range mi.SentReqs {
+					// Mock database implementation will only return first
+					// replica so make sure the hostlist sent in pool rebuild
+					// manage requests matches what leader query returns as the
+					// first replica.
```

Comment:
Is it possible to fix this? Seems like in this case we'd really benefit from having the full list.

Reply:
Ah, okay. Fair point.